### ISSUE TITLE
Week of 9/19/2023

### ISSUE THREAD

**Issue by glenncampagna** [HARPgroup/HARParchive#1101]
- [ ] #1092
    - Water availability article: [Estimating 7Q10 confidence limits from data: a bootstrap approach DP Ames - Journal of water resources planning and management, 2006 - ascelibrary.org](https://ascelibrary.org/doi/10.1061/%28ASCE%290733-9496%282006%29132%3A3%28204%29) --> see who’s cited it, and summarize how folks are calculating uncertainty for streamflow records
    - [water stats guy website](https://www.usgs.gov/staff-profiles/robert-m-hirsch)
    - Goal: we want to know how much uncertainty and variation our calculations will have. How reliable is our calculation?
- [ ] Continue working to turn what we have into a full document
    - [ ] update the detailed work plan to show more of what we have, get questions that we can ask soon before new generation
    - [ ] edit for using new foundation file
- [ ] Tasks for another om pull request laid out in https://github.com/HARPgroup/HARParchive/issues/818#issuecomment-1725373771
  - How can I use `sudo` to test the scripts once merged into the om repo?
- [ ] RB/JK - build plumbing to generate and stash usgs model metrics
  - [ ] Use New Markdown gage comparison script, but amend it to:
    - [ ] Look on the river feature for a usgs weighted model with `propcode = usgs-1.0`
    - [ ] load the USGS model if it exists, or create it if it does not
    - [ ] store values calculated for metrics L90, etc. in scenario metrics for 
    - [ ] store `area_factor` in model scenario metrics
    - [ ] store `usgs_gageid` in model scenario metrics
    - [ ] if user passes in 'area_factor=auto`, try to load the vahydro model and get the drainage area, and load the drainage area from the USGS gage and calculate an area_factor (model drainage area / gage drainage area)

**Comment by glenncampagna** [comment]
### Questions for @rburghol and @jdkleiner 
- Does `area_factor` need to be applied to the gage metrics we're comparing to model metrics in `compare_model_gage_metrics.R`? It looks like historic gage discharge data is multiplied by the area_factor in `gage_vs_model.Rmd` ... should the gage metrics in this case be multiplied by their gage's area factor? Or are the gage metrics already 'weighted' with the area factor?
- Is the pull request open on the om repo ready to be merged? I saw the approval from Rob but don't want to jump the gun. I've documented the changes to be made in an additional PR I'll be working on this week 
- Once the PR is merged to the om repo, I'd like to do a final test of the summary scripts and function. We talked at the end of the summer about this and I was told that I can either `sudo` to get around permissions or have DEQ do some tests. Is the `sudo` process I'd use simple enough to explain in a comment? If not I can wait until we meet next
